tests/conformance/state/state.go

```diff
@@ -300,10 +365,12 @@ func ConformanceTests(t *testing.T, props map[string]string, statestore state.St
 			// Check if query feature is listed
 			features := statestore.Features()
 			require.True(t, state.FeatureQueryAPI.IsPresent(features))
-
 			querier, ok := statestore.(state.Querier)
 			assert.True(t, ok, "Querier interface is not implemented")
 			for _, scenario := range queryScenarios {
+				if (scenario.partitionOnly) && (!state.FeaturePartitionKey.IsPresent(features)) {
+					break
+				}
```

Comment:
@berndverst the test will be executed only for store with partitionKey. We can remove the test with partition key or if you can suggest me some solution to execute this test without using the new feature PartitionKey.